tripl/tripl.py

```diff
@@ -526,8 +526,8 @@ def pull_many(self, pull_expr, eids_or_pattern, sort_by=None, sort_desc=True):
 def entity_cons(type_name, default_attr_base):
     "Return a constructor function for creating namespaced entities"
     def f(**avs):
-        avs = dict(((default_attr_base + ':' + k if ':' not in k else k), v) for k, v in avs.iteritems())
-        avs['cft:type'] = type_name
+        avs = dict(((default_attr_base + ':' + k if ':' not in k else k), v) for k, v in avs.items())
+        avs[type_name.split('.')[0] + ':type'] = type_name
```

Comment:
Good catch! Actually though, it might be better to have the type attribute name as either part of the schema, or just just a generic thing like `tripl.entity:type`. For now I'll just merge, but I'll add this idea as an issue.